src/Components/ExtraInfo/index.jsx

```diff
@@ -0,0 +1,204 @@
+import React, { useEffect, useState } from 'react'
+import useAuth from '../../Context'
+import { Maps } from '../Maps'
+
+export const ExtraInfo = (props) => {
+  const [savedEventsArray, setsavedEventsArray] = useState([])
+  const { logout } = useAuth()
+  const { userSpotifyID } = useAuth()
+
+  useEffect(() => {
+    fetchSavedEvents()
+  }, [])
+
+  const fetchSavedEvents = async () => {
+    const result = await fetch(
+      `http://localhost:3001/events/${userSpotifyID}`,
+      {
+        method: 'GET',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      }
+    )
+    const response = await result.json()
+    const eventIDs = response.map((event, index) => {
+      return event.event_id
+    })
+    console.log('eventIDs', eventIDs)
+    setsavedEventsArray(eventIDs)
+  }
+
+  const handleClick = async (eventID, event_name, date, city, url) => {
+    console.log('clicked')
+    if (savedEventsArray.includes(eventID)) {
+      try {
+        const data = {
+          event_id: eventID,
+        }
+        const result = await fetch(
+          `http://localhost:3001/events/${userSpotifyID}`,
```

Comment:
if you deploy this it won't work as it's going to be looking for localhost and not the correct url

Reply:
changed to env link now